guide/ops/requirements.md

```diff
@@ -19,6 +19,43 @@ at least 8GB RAM and 100GB disk. The disk is just for logs, a small amount of pe
 any binaries for custom blueprints/integrations.
 
 
+### Disk Space
+
+There are three main consumers of disk space:
+
+* **Static files**: these are the Apache Brooklyn files themselves, plus
```

Comment:
Perhaps reword: the installation and dependencies (e.g. Java). Therefore it would be applicable to RPM installs as well.

Separate point for blueprints added to the catalog. These can include: OSGi bundles for Java-based blueprints or associated resources; and/or jars added to the `lib/dropins/` directory for old-style Java-based blueprints.